lib/wrappers/angular.js

```diff
@@ -27,8 +27,14 @@
             }, angularAdapter);
 
             // Wrap `execute()` in an Angular promise
-            api.execute = function(method, params) {
-                return $q.when(TaggedApi.prototype.execute.call(this, method, params));
+            api.execute = function(method, options) {
+                if (!options.clientId || typeof options.clientId === 'undefined') {
+                    return $q.reject('Client ID is required to make API calls');
+                }
+
+                // append the client ID from the params sent from client
+                TaggedApi.prototype.execute.call(this, options.clientId);
```

Comment:
This doesn't appear to be a valid call. `TaggedApi.prototype.execute` expects the first parameter to be an API method name, e.g. `tagged.log.click`. It doesn't make sense to pass in client id here.